src/imgen.c

```diff
@@ -13,6 +13,11 @@
 #define STB_IMAGE_WRITE_IMPLEMENTATION
 #include <stb/stb_image_write.h>
 
+// default image width, height
+#define HEIGHT 1080
+#define WIDTH 1080
```

Comment:
Is this the size of the image displayed in pixels?

Reply:
Yes